[CI] VectorScorerFactoryTests testRandomScorerMax failing

**Build Scans:**
- [elasticsearch-intake #21801 / part1](https://gradle-enterprise.elastic.co/s/azzbqovjgt5pg)
- [elasticsearch-pull-request #67857 / part-1](https://gradle-enterprise.elastic.co/s/vl56ws4pbgura)

**Reproduction Line:**
```
./gradlew ":libs:simdvec:test" --tests "org.elasticsearch.simdvec.VectorScorerFactoryTests.testRandomScorerMax" -Dtests.seed=7FDA25580A95E52C -Dtests.locale=en-KN -Dtests.timezone=Africa/Bissau -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.simdvec.VectorScorerFactoryTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testRandomScorerMax),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: expected:<743.881> but was:<743.88086>
```

**Issue Reasons:**
- [main] 2 failures in test testRandomScorerMax (1.1% fail rate in 189 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/libs/simdvec/src/test/java/org/elasticsearch/simdvec/VectorScorerFactoryTests.java
+++ b/libs/simdvec/src/test/java/org/elasticsearch/simdvec/VectorScorerFactoryTests.java
@@ -50,8 +50,6 @@ import static org.hamcrest.Matchers.greaterThanOrEqualTo;
 // @com.carrotsearch.randomizedtesting.annotations.Repeat(iterations = 100)
 public class VectorScorerFactoryTests extends AbstractVectorTestCase {
 
-    private static final float DELTA = 1e-4f;
-
     // bounds of the range of values that can be seen by int7 scalar quantized vectors
     static final byte MIN_INT7_VALUE = 0;
     static final byte MAX_INT7_VALUE = 127;
@@ -260,6 +258,8 @@ public class VectorScorerFactoryTests extends AbstractVectorTestCase {
                 final byte[][] qVectors = new byte[size][];
                 final float[] corrections = new float[size];
 
+                float delta = 1e-6f * dims;
+
                 String fileName = "testRandom-" + sim + "-" + dims + ".vex";
                 logger.info("Testing " + fileName);
                 try (IndexOutput out = dir.createOutput(fileName, IOContext.DEFAULT)) {
@@ -280,7 +280,7 @@ public class VectorScorerFactoryTests extends AbstractVectorTestCase {
 
                         var expected = luceneScore(sim, qVectors[idx0], qVectors[idx1], correction, corrections[idx0], corrections[idx1]);
                         var scorer = factory.getInt7SQVectorScorer(VectorSimilarityType.of(sim), values, vectors[idx0]).get();
-                        assertEquals(scorer.score(idx1), expected, DELTA);
+                        assertEquals(scorer.score(idx1), expected, delta);
                     }
                 }
             }

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -396,18 +396,12 @@ tests:
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=transform/transforms_start_stop/Test start/stop/start continuous transform}
   issue: https://github.com/elastic/elasticsearch/issues/126755
-- class: org.elasticsearch.simdvec.VectorScorerFactoryTests
-  method: testRandomScorerMax
-  issue: https://github.com/elastic/elasticsearch/issues/126797
 - class: org.elasticsearch.search.SearchServiceSingleNodeTests
   method: testBeforeShardLockDuringShardCreate
   issue: https://github.com/elastic/elasticsearch/issues/126812
 - class: org.elasticsearch.search.SearchServiceSingleNodeTests
   method: testLookUpSearchContext
   issue: https://github.com/elastic/elasticsearch/issues/126813
-- class: org.elasticsearch.simdvec.VectorScorerFactoryTests
-  method: testRandomScorerChunkSizeSmall
-  issue: https://github.com/elastic/elasticsearch/issues/126847
 
 # Examples:
 #

